src/sonic-py-common/sonic_py_common/logger.py

```diff
@@ -11,30 +11,35 @@ class Logger(object):
     """
     Logger class for SONiC Python applications
     """
-    LOG_FACILITY_USER = syslog.LOG_USER
     LOG_FACILITY_DAEMON = syslog.LOG_DAEMON
+    LOG_FACILITY_USER = syslog.LOG_USER
+
+    LOG_OPTION_NDELAY = syslog.LOG_NDELAY
+    LOG_OPTION_PID = syslog.LOG_PID
 
     LOG_PRIORITY_ERROR = syslog.LOG_ERR
     LOG_PRIORITY_WARNING = syslog.LOG_WARNING
     LOG_PRIORITY_NOTICE = syslog.LOG_NOTICE
     LOG_PRIORITY_INFO = syslog.LOG_INFO
     LOG_PRIORITY_DEBUG = syslog.LOG_DEBUG
 
-    def __init__(self, log_identifier=None, log_facility=LOG_FACILITY_USER):
-        self.syslog = syslog
+    DEFAULT_LOG_FACILITY = syslog.LOG_USER
+    DEFAULT_LOG_OPTION = syslog.LOG_NDELAY
```

Comment:
Suggest using the constants defined above, e.g.,

```python
    DEFAULT_LOG_FACILITY = self.LOG_FACILITY_USER
    DEFAULT_LOG_OPTION = self.LOG_OPTION_NDELAY
```

Reply:
@jleveque fixed

Reply:
Do you not need `self.` here?